src/ProjectTemplates/test/SpaTemplateTest/AngularTemplateTest.cs

```diff
@@ -17,8 +17,13 @@ public AngularTemplateTest(ProjectFactoryFixture projectFactory, BrowserFixture
             : base(projectFactory, browserFixture, output) { }
 
         [Fact]
-        public Task AngularTemplate_Works()
```

Comment:
Next time, just add [Repeat(10)] 😉 .

As long as you revert this after it all passes, 🚢 🇮🇹